lib/src/internet_connection.dart

```diff
@@ -127,11 +127,19 @@ class InternetConnection {
     InternetCheckOption option,
   ) async {
     try {
-      final response = await http.head(option.uri).timeout(option.timeout);
+      final response = await http
+          .head(
+            option.uri,
+            headers: option.headers,
```

Comment:
Please remove the trailing comma on this line and run:

```bash
dart format .
```

Reply:
Sorry, I was used to use tralling comma. If its not the style, I'll remove it.

Reply:
i've reformat it, please review it again